insights/client/utilities.py

```diff
@@ -124,7 +124,10 @@ def write_to_disk(filename, delete=False, content=get_time()):
         return
     if delete:
         if os.path.lexists(filename):
-            os.remove(filename)
+            try:
+                os.remove(filename)
+            except OSError:
+                return
```

Comment:
As OSError could have some other causes than the file not existing, I will just check for the errno == 2.
That way it will not produce an error when it doesn't exists but it will do it if anything else happens.

Reply:
Good point, would be safer to be specific with a `FileNotFoundError` here.

Reply:
Actually,  this won't work. Nevermind, `FileNotFoundError` isn't a thing in python2.6/2.7....